src/MonoMod.UnitTest/TestLogHelpers.cs

```diff
@@ -19,7 +19,9 @@ internal static class TestLogHelpers {
                         return;
                 }
                 var localTime = time.ToLocalTime();
+#pragma warning disable CA1305 // Specify IFormatProvider
                 helper.WriteLine($"[{source}]({localTime}) {level.FastToString()}: {message}");
+#pragma warning restore CA1305 // Specify IFormatProvider
```

Comment:
Same as above. For this though, I'd rather just explicitly pass `null` to `FastToString`.